index.js

```diff
@@ -63,17 +63,17 @@ export type Options = {
  * on the current device.
  * @return {Promise} Resolves to `SECURITY_LEVEL` when supported, otherwise `null`.
  */
-export function getSecurityLevel(): Promise<?($Values<typeof SECURITY_LEVEL>)> {
+export function getSecurityLevel(options?: Options): Promise<?($Values<typeof SECURITY_LEVEL>)> {
     if (!RNKeychainManager.getSecurityLevel){
         return Promise.resolve(null);
     }
-    return RNKeychainManager.getSecurityLevel();
+    return RNKeychainManager.getSecurityLevel(options?: Options);
```

Comment:
There seems to still be a small merge-problem here, with types being included to the function call argument.